xscen/ensembles.py

```diff
@@ -1,24 +1,28 @@
 import logging
 from pathlib import Path
-from typing import Optional, Union
 
 import pandas as pd
 import xarray as xr
 from xclim import ensembles
 
-from .catalog import ProjectCatalog, generate_id
+from xscen.catalog import generate_id  # ProjectCatalog
```

Comment:
```suggestion
from .catalog import generate_id  # ProjectCatalog
```
pourquoi  mettre xscen ?

Reply:
Artefact from refactoring. Good to remove.